pkgs/development/python-modules/python-chess/default.nix

```
@@ -0,0 +1,26 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pythonOlder
+}:
+
+buildPythonPackage rec {
+  pname = "python-chess";
+  version = "1.6.1";
+
+  disabled = pythonOlder "3.7";
+
+  src = fetchPypi {
+    inherit version;
+    pname = "chess";
+    sha256 = "sha256-lmumSDVZ3A+fOr1vryvKLhBVBGavQjvnmodlY509zTM=";
+  };
+
+  doCheck = false;
+
+  meta = with lib; {
+    description = "A chess library for Python, with move generation, move validation, and support for common formats.";
+    homepage = "https://github.com/niklasf/python-chess";
+    license = licenses.gpl3Plus;
```

New packages need a maintainer.

```suggestion
    license = licenses.gpl3Plus;
    maintainers = with maintainers; [ smancill ];
```

---

Added myself as maintainer. I didn't know if I could just add my name to the list.